source/localizable/tutorial/autocomplete-component.md

```diff
@@ -1,100 +1,173 @@
-As they search for a rental, users might also want to narrow their search
-to a specific city. Let's build a component that will let them search for
-properties within a city, and also suggest cities to them as they type.
+As they search for a rental, users might also want to narrow their search to a specific city. 
+Let's build a component that will let them filter rentals by city.
 
-To begin, let's generate our new component. We'll call this component
-`filter-listing`.
+To begin, let's generate our new component. 
+We'll call this component `list-filter`, since all we want our component to do is filter the list of rentals based on input.
 
 ```shell
-ember g component filter-listing
+ember g component list-filter
 ```
 
-As before, this creates a Handlebars template
-(`app/templates/components/filter-listing.hbs`) and a JavaScript file
-(`app/components/filter-listing.js`).
+As before, this creates a Handlebars template (`app/templates/components/list-filter.hbs`), 
+a JavaScript file (`app/components/list-filter.js`), 
+and a component integration test (`tests/integration/components/list-filter-test.js`).
 
-The Handlebars template looks like this:
+Let's start with writing some tests to help us think through what we are doing.
+The filter component should yield a list of filtered items to its inner template block.
```

Comment:
filtered items to _whatever renders inside of it, known as its inner template block_